Fix `docker-compose ps` output (#181)
grafana/tutorials: content/tutorials/grafana-fundamentals-cloud.md

1. For **Section 2**, find the **query A** box. Choose your Prometheus datasource and enter the same query that we used in our earlier panel: `sum(rate(tns_request_duration_seconds_count[5m])) by(route)`. Press **Run queries**. You should see some data in the graph.
1. Now scroll down to the **query B** box. For **Operation** choose `Classic condition`. [You can read more about classic and multi-dimensional conditions here](https://grafana.com/docs/grafana/latest/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule/#single-and-multi-dimensional-rule). For conditions enter the following: `WHEN last() OF A IS ABOVE 0.2`
1. In **Section 3**, enter `30s` for the **Evaluate every** field. For the purposes of this tutorial, the evaluation interval is intentionally short. This makes it easier to test. In the **For** field, enter `0m`. This setting makes Grafana wait until an alert has fired for a given time before Grafana sends the notification.

> 1. In **Section 4**, you can add some sample text to your summary message. [Read more about message templating here](https://grafana.com/docs/grafana/next/alerting/contact-points/message-templating/).

Reviewer: This doc is published to `/latest/`.

```suggestion
1. In **Section 4**, you can add some sample text to your summary message. For more information about message templating, refer to the [documentation](https://grafana.com/docs/grafana/latest/alerting/contact-points/message-templating/).
```
